lib/index.js

```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 const emojer = require("emojer")
-    , emojis = require("github-api-emojis/lib/map")
+    , emojis = require("github-api-emojis/lib/map.json")
```

Comment:
Why is this needed?

Reply:
We use webpack-encore. When run the bellow command, the system doesn't find the dependency.

# ./node_modules/.bin/encore dev
Running webpack ...

This dependency was not found:

* github-api-emojis/lib/map in ./node_modules/showdown-emoji/lib/index.js

To install it, you can run: npm install --save github-api-emojis/lib/map

Reply:
Umm, well, that seems to be more like a webpack-encore issue, then. I would suggest opening an issue in their repository as well.

If adding `.json`solves the issue here, I'll gladly merge the PR. 😁

Reply:
Thank you.